codebuild/bin/s2n_setup_env.sh

```diff
@@ -172,6 +172,30 @@ for i in $path_overrides; do testdeps_path "$i" ;done
 if [ -f "/etc/lsb-release" ]; then
   cat /etc/lsb-release
 fi
+
+# Translate our custom variables into full paths to the compiler.
+set_cc(){
+  if [ -z ${GCC_VERSION:-} -o ${GCC_VERSION} = "NONE" ]; then
+    echo "No GCC_VERSION set"
+    if [ ${LATEST_CLANG:-} = "true" ]; then
+      echo "LATEST_CLANG is ${LATEST_CLANG}"
+      if [ -d ${LATEST_CLANG_INSTALL_DIR:-} ]; then
+        export CC=${LATEST_CLANG_INSTALL_DIR}/bin/clang
+        echo "CC set to ${CC}"
```

Comment:
should we not also set CXX to clang++ here?

Reply:
yes